packages/neutrino-middleware-eslint/index.js

```diff
@@ -24,10 +26,10 @@ const getEslintRcConfig = pipe(
 );
 
 module.exports = (neutrino, opts = {}) => {
-  const isNotDev = process.env.NODE_ENV !== 'development';
+  const failBuild = process.env.NODE_ENV !== 'development' || neutrino.options.command !== 'start';
   const options = merge.all([
     opts,
-    !opts.include && !opts.exclude ? { include: [neutrino.options.source], exclude: [neutrino.options.static] } : {}
+    !opts.include && !opts.exclude ? { include: [neutrino.options.source] } : {}
```

Comment:
Should this now be:
`!opts.include ? { include: [neutrino.options.source] } : {}`

This would allow people to specify an exclude list, but keep the default include, saving having to pass it unnecessarily (Treeherder hits this case for example).

Reply:
Yeah, good catch.